embedded-couchbase/src/test/java/com/playtika/test/couchbase/EmbeddedCouchbaseBootstrapConfigurationTest.java

```diff
@@ -12,6 +12,7 @@
         classes = {CouchbaseConfiguration.class},
         properties = {
                 "spring.profiles.active=enabled",
+                "spring.main.allow-bean-definition-overriding=true",
```

Comment:
`TransactionInterceptor` in `AbstractCouchbaseConfiguration` overrides `TransactionInterceptor` in `ProxyTransactionManagementConfiguration`.